mesheryctl/internal/cli/root/system/model.go

```diff
@@ -413,6 +426,120 @@ mesheryctl exp model search [query-text]
 	},
 }
 
+// add command to push aritfacts to the oci registry
+var pushModelCmd = &cobra.Command{
+	Use:   "push",
+	Short: "push model",
+	Long:  "push a model to the registry",
+	Example: `
+// Push a model
+mesheryctl exp model push [path-to-model] --username [username] --password [password] --registry [registry] --tag [tag] --repository [repository]
+	`,
+	// skip preRunE as it is not required for this command
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) == 0 {
+			return cmd.Help()
+		}
+
+		// path to model
+		modelPath := args[0]
+
+		// Push logic
+		fs, err := file.New(".")
+		if err != nil {
+			return err
+		}
+		ctx := context.Background()
+
+		mediaType := "application/vnd.test.folder"
+		fileNames := []string{modelPath}
+		fileDescriptors := make([]v1.Descriptor, 0)
+		for _, name := range fileNames {
+			fileDescriptor, err := fs.Add(ctx, name, mediaType, "")
+			if err != nil {
+				panic(err)
+			}
+			fileDescriptors = append(fileDescriptors, fileDescriptor)
+			fmt.Printf("file descriptor for %s: %v\n", name, fileDescriptor)
```

Comment:
meshkit logger.

Reply:
@RipulHandoo feedback from two weeks ago is ready for incorporation.